source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt

```diff
@@ -4,23 +4,24 @@
 Write Clear and Concise Sentences and Paragraphs
 ================================================
 
+.. default-domain:: mongodb
+
 .. include:: /includes/styles/corrections.rst
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
 Users can understand wordy and complex writing. They can wade through
 walls of text, but we shouldn't force them to do that. Use the
-following guidelines to help you write clear and concise sentences and
-paragraphs.
+following guidelines to clarify and condense sentences and paragraphs.
 
 .. _use-consistent-sentence-structure:
 
 Use a Consistent Sentence Structure
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------------------
```

Comment:
General rule of thumb: don't force users to backtrack when reading a sentence. If they have to gather necessary details at the end to understand the beginning, see if you can rewrite it.

Example: 

"Gathering the data necessary to parse dense content effectively is a task struggled with by many users"

Is basically about as backwards as you can get. ** shudders **